discussion/src/main/java/org/openedx/discussion/presentation/ui/DiscussionUI.kt

```diff
@@ -312,6 +314,7 @@ fun CommentItem(
                         color = MaterialTheme.appColors.textPrimaryVariant
                     )
                 }
+                Spacer(Modifier.weight(1f))
```

Comment:
The issue with this is that if the author's name is big, then the button will be pushed out of the view frame. Same goes for the other `weight(1f)`